docs/devices.rst

```diff
@@ -0,0 +1,147 @@
+    >>> job = query.export()
+    >>> csv_report = job.get_output_as_string()
+    >>> # can also get the output as a file or as enumerated lines of text
+
+Search Scrolling
+++++++++++++++++
+
+A Device Search request can return no more than 10,000 items at a time.  Some customers may have more endpoints than
+that; to return *all* devices, you can use the ``scroll()`` method on the query to continue searching after all devices
+that have been previously returned.  This snippet illustrates the technique::
+
+    # assume "api" is your CBCloudAPI reference
+    query = api.select(Device)
+    # add search terms and/or criteria to the query (not shown here)
+    for d in query:
+        do_something_with_device(d)  # whatever you need for each device
+    while query.num_remaining > 0:
+        query.scroll()  # default is 10,000 devices at a time
+        for d in query:
+            do_something_with_device(d)
+
+Device Actions
+--------------
+
+Most device actions in the Carbon Black Cloud can be performed on a single device through the ``Device`` object,
+on multiple devices specified by ID, or on the results of a device query.
+
+Bypass Enable/Disable
++++++++++++++++++++++
+
+Setting a device to *bypass* disables all enforcement on the device; its sensor stops sending data to the Carbon Black
+Cloud.
+
+Setting bypass on a single device::
+
+    >>> # assume "api" is your CBCloudAPI reference
+    >>> d = api.select(Device, 12345)
+    >>> d.bypass(True)
+
+Setting bypass on multiple devices::
+
+    >>> # assume "api" is your CBCloudAPI reference
+    api.device_bypass([1001, 1002, 1003], True)
+
+Setting bypass on the results of a device search::
+
+    >>> # assume "api" is your CBCloudAPI reference
+    query = api.select(Device)
+    # add search terms and/or criteria to the query (not shown here)
+    query.bypass(True)
+
+Quarantine
+++++++++++
+
+A device that has been *quarantined* has its outbound traffic limited, and all inbound traffic to it stopped.  This
+would be used on any device determined to be interacting badly.
+
+Setting quarantine on a single device::
+
+    >>> # assume "api" is your CBCloudAPI reference
+    >>> d = api.select(Device, 12345)
+    >>> d.quarantine(True)
+
+Setting quarantine on multiple devices::
+
+    >>> # assume "api" is your CBCloudAPI reference
+    api.device_quarantine([1001, 1002, 1003], True)
+
+Setting quarantine on the results of a device search::
+
+    >>> # assume "api" is your CBCloudAPI reference
+    query = api.select(Device)
+    # add search terms and/or criteria to the query (not shown here)
+    query.quarantine(True)
+
+Background Scan
++++++++++++++++
+
+Enabling *background scan* causes a one-time inventory scan on the device to identify any malware files already present
+there.  The background scan is of the type specified in the device's policy, if that policy has background scans
+enabled, or a standard background scan if it does not.
```

Comment:
> The background scan is of the type specified in the device's policy, if that policy has background scans
enabled, or a standard background scan if it does not.

Not sure what you mean by background scan type. Either the policy indicates that the scan will be performed on sensor installation or can be manually triggered at a later point.

All backgrounds scans are the same

Reply:
Again, that's straight from the user guide. Maybe the explanation there was unclear.